src/main/java/uk/gov/ida/eventemitter/EventEmitter.java

```diff
@@ -5,17 +5,28 @@
 public class EventEmitter {
 
     private final boolean sendToRecordingSystem;
+    private final Encrypter encrypter;
+    private final SqsClient sqsClient;
 
     @Inject
-    public EventEmitter(boolean sendToRecordingSystem) {
+    public EventEmitter(final boolean sendToRecordingSystem,
+                        final Encrypter encrypter,
+                        final SqsClient sqsClient) {
         this.sendToRecordingSystem = sendToRecordingSystem;
+        this.encrypter = encrypter;
+        this.sqsClient = sqsClient;
     }
 
-    public void record(Event event) {
+    public void record(final Event event) {
         if (sendToRecordingSystem) {
```

Comment:
We shouldn't need this if statement any more.
If we aren't sending, we can just be using the stub sqs client, and everything is nice and same.

Reply:
Sounds good. I have removed SendToRecordingSystem.